test/routes/healthcheck.test.js

```diff
@@ -0,0 +1,24 @@
+const request = require('supertest');
+const createServer = require('../helpers/create-server');
+
+const ACCESS_CONTROL_ALLOW_ORIGIN = 'access-control-allow-origin';
+const envSecret = Array(65).join('0');
+const authSecret = Array(65).join('1');
+
+describe('routes > healthcheck', () => {
+  describe('#GET /forest/healthcheck', () => {
+    it('should return 200', async () => {
+      expect.assertions(2);
+      const app = createServer(envSecret, authSecret);
+      await new Promise((done) => {
+        request(app)
+          .get('/forest/healthcheck')
+          .end((error, response) => {
+            expect(response.status).toStrictEqual(200);
+            expect(response.res.headers[ACCESS_CONTROL_ALLOW_ORIGIN]).toStrictEqual('*');
```

Comment:
If it is a simple request you won't expect this header.

Reply:
Answered above: we need it because without it the browser doesn't let access to the response code.